atd-vzd/schema/atd_txdot_person.sql

```diff
@@ -87,69 +87,147 @@ CREATE SEQUENCE public.atd_txdot_person_person_id_seq
 --
 
 CREATE INDEX atd_txdot_person_is_retired_index ON public.atd_txdot_person USING btree (is_retired);
 
 
 --
--- TOC entry 5622 (class 1259 OID 2346776)
--- Name: atd_txdot_person_person_id_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_person_person_id_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_person_person_id_index ON public.atd_txdot_person USING btree (person_id);
 
 
 --
--- TOC entry 5623 (class 1259 OID 2346796)
--- Name: idx_atd_txdot_person_crash_id; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_person_prsn_age_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_age_index ON public.atd_txdot_person USING btree (prsn_age);
+
+
+--
+-- Name: atd_txdot_person_prsn_death_date_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_death_date_index ON public.atd_txdot_person USING btree (prsn_death_date);
+
+
+--
+-- Name: atd_txdot_person_prsn_death_time_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_death_time_index ON public.atd_txdot_person USING btree (prsn_death_time);
+
+
+--
+-- Name: atd_txdot_person_prsn_ethnicity_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_ethnicity_id_index ON public.atd_txdot_person USING btree (prsn_ethnicity_id);
+
+
+--
+-- Name: atd_txdot_person_prsn_gndr_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_gndr_id_index ON public.atd_txdot_person USING btree (prsn_gndr_id);
+
+
+--
+-- Name: atd_txdot_person_prsn_injry_sev_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_prsn_injry_sev_id_index ON public.atd_txdot_person USING btree (prsn_injry_sev_id);
+
+
+--
+-- Name: atd_txdot_person_sus_serious_injry_cnt_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_person_sus_serious_injry_cnt_index ON public.atd_txdot_person USING btree (sus_serious_injry_cnt);
+
+
+--
+-- Name: idx_atd_txdot_person_crash_id; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX idx_atd_txdot_person_crash_id ON public.atd_txdot_person USING btree (crash_id);
 
 
 --
--- TOC entry 5625 (class 2620 OID 2346816)
--- Name: atd_txdot_person atd_txdot_person_audit_log; Type: TRIGGER; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_person atd_txdot_person_audit_log; Type: TRIGGER; Schema: public; Owner: visionzero
 --
 
-CREATE TRIGGER atd_txdot_person_audit_log BEFORE UPDATE ON public.atd_txdot_person FOR EACH ROW EXECUTE PROCEDURE public.atd_txdot_person_updates_audit_log();
+CREATE TRIGGER atd_txdot_person_audit_log BEFORE UPDATE ON public.atd_txdot_person FOR EACH ROW EXECUTE FUNCTION public.atd_txdot_person_updates_audit_log();
+
+ALTER TABLE public.atd_txdot_person DISABLE TRIGGER atd_txdot_person_audit_log;
```

Comment:
Not related to your changes, but it is interesting that we have this trigger defined and then immediately disabled here. 

Makes me wonder if this is more cruft that @frankhereford would like a look at. Maybe we already know about this or there is more to it that I'm seeing.

Reply:
Good question! I wonder if this SQL was taken from a `pg_dump` which will embed the enabling and disabling of triggers as needed so that the ultimate final result after the entire output is what the DB was when the export was made. Like, disabling triggers before all the `insert`s and then re-enabling them. 

In this case, I think it will leave the database with that trigger disabled, so this line should probably be removed. Does that seem right @roseeichelmann?

Reply:
honestly im not sure, i just did pg_dumps of these tables bc i saw they havent been updated in a long time and figured i might as well. now im thinking we should do pg_dumps of the production db and commit them after the release of this feature and all the needed db changes in prod are made. idk why my `pg_dump` would disable this trigger though?

Reply:
I think what happens is that `pg_dump` is working in logical phases, sort of like this, because it's operating in a bigger context of "my job is to create the whole DB in a certain state from whole cloth."

* Enter into create the schema mode
  * Create the table
  * Setup indexes, constraints, triggers, etc for the table
* Exit create schema mode
* Enter into populate data mode
  * Disable triggers
  * Insert the data, without the triggers firing for each record
  * Re-enable triggers
* Exit populate data mode

I think when you captured these files out of `pg_dump`'s output, you may have caught the first part of populate data mode, which has that disable statement. I think you can cut off where you're pulling from the `pg_dump` a little earlier and be good. I'd be super happy to sync if that would be helpful - I am probably explaining this poorly.